src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -50,6 +50,16 @@
  * <pre>
  * &lt;module name="JavadocTagContinuationIndentation"/&gt;
  * </pre>
+ * <pre>
+ * &#47;**
+ *  * @tag comment..
+ *  *  Indentation spacing is 1 (violation)
```

Comment:
`(violation)` -> // violation 

Here and in other cases below

Reply:
@strkkk  examples of other Javadoc checks have the format (`violation)`....should I still change it to `// violation` ?

Reply:
What checks do you mean?

Most of checks have this format `// violation`, so it is better to keep it consistent

Reply:
![jdtc4](https://user-images.githubusercontent.com/38028330/79370062-a7b55e00-7f6f-11ea-9f55-8c92785dccf9.png)


@strkkk Javadoc Comments have this format

Reply:
yes, but some have no format at all...

Lets do it like in most other javadoc checks, like
```
/**
* Line with violation
* OK line
*/
```

Reply:
Ok, done